test/AutoDiff/compiler_crashers_fixed/issue-62608-conflicting-debug-info.swift

```diff
@@ -0,0 +1,79 @@
+// RUN: %target-swift-frontend -emit-sil -O -g %s | %FileCheck %s
+
+// REQUIRES: swift_in_compiler
```

Comment:
Why does this require `swift_in_compiler`?

Reply:
The code is optimized on Windows differently due to lack of optimization passes written in Swift....